dproc/dproc_test.go

```diff
@@ -108,13 +112,13 @@ func TestGetDataprocRequestParams(t *testing.T) {
 			errorExpected: false,
 		},
 		{
-			name: "valid mysql source profile and target profile with missing primary key in source",
```

Comment:
is the mysql test being replaced with the postgres test here ?

Reply:
MySQL test for missing primary key got lost on remote branch merge. 
Fixed it , Commit: https://github.com/GoogleCloudPlatform/spanner-migration-tool/pull/594/commits/2caf17d33348b36cab183961f592bd52f878283d